runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1891,6 +1902,32 @@ class INTERPRETER_CLASS
 				}
 				break;
 			}
+#if defined(J9VM_OPT_MICROJIT)
+			if (mjitExtra) {
+				if (J9_ARE_NO_BITS_SET(mjitExtra, J9_STARTPC_NOT_TRANSLATED)) {
```

Comment:
Won't the `break` in the code above - `if (J9_ARE_NO_BITS_SET(preCount, J9_STARTPC_NOT_TRANSLATED)) ` prevent us from ever getting here?

Reply:
I believe you are correct and this is another missed piece of code that should have been removed when `extra2` was removed.